swap/swap_test.go

```diff
@@ -1029,7 +1026,7 @@ func testWaitForTx(auth *bind.TransactOpts, backend cswap.Backend, tx *types.Tra
 // deploy for testing (needs simulated backend commit)
 func testDeploy(ctx context.Context, swap *Swap) (err error) {
 	opts := bind.NewKeyedTransactor(swap.owner.privateKey)
-	opts.Value = big.NewInt(int64(swap.params.InitialDepositAmount))
+	opts.Value = big.NewInt(int64(0))
```

Comment:
i know the value is `9999999999` now, but can you clarify this change? why can we not test with that one? 

or: if we need to test with `0`, why is the initial value `9999999999`?

Reply:
Thanks for this comment! It made it obvious we had a problem here.
I changed this because tests were failing with `InsufficientFunds`, but my conclusion was wrong. 

I reduced the initial deposit amount for tests, and then `TestContractIntegration` was failing: because it was built with the previous expectation that the initial amount was 0 -> therefore it did a manual TX to fund the chequebook causing now too much funds in the contract, and the cheque expecting to bounce was not bouncing anymore.

Fixed by removing the manual funding TX in `TestContractIntegration`